docs/css/custom.css

```diff
@@ -618,3 +618,13 @@ hr {
     left: auto;
   }
 }
+
+/* Definition from original RTD theme, add here to extend to sidebar links */
+.wy-menu-vertical li.toctree-l1 a.reference.external:after {
+  font-family: FontAwesome;
+  content: "";
+  color: #b3b3b3;
+  vertical-align: super;
```

Comment:
I think this looks nicer when set to `middle`

Reply:
Good question whether that is fixable. It's auto-generating that, so maybe we've got a weird link in the RST. I can check that tomorrow.

As far as looking nicer with middle, I'm fine with whatever, I just copied the style out of what was currently there. So if we switch my new class to `middle`, we should change the other one as well?